addons/l10n_it/migrations/15.0.0.3/post-migrate.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from odoo import api, SUPERUSER_ID
+
+def migrate(cr, version):
+
+    env = api.Environment(cr, SUPERUSER_ID,{})
+    
+    cr.execute("""
+        INSERT INTO account_account_account_tag
+        SELECT a.id, att.account_account_tag_id
+        FROM account_account AS a, account_account_template_account_tag AS att,
+        account_account_template AS at
+        WHERE a.code LIKE CONCAT(at.code,'%')
+        AND at.id = att.account_account_template_id
```

Comment:
What about multicompany/multi-l10n ? This will target ALL accounts and ALL templates ! You should restrict this to the templates from l10n_it. You can easily do this by joining the account_account_template table with ir_model_data and checking the module in which the ir_model_data is defined ;)